package.json

```diff
@@ -18,11 +18,13 @@
     "update-version-patch": "npm version patch",
     "test": "mocha -r ts-node/register test/**/*.spec.ts",
     "test-smart-contract-example": "ts-node ./examples/smartContracts/index.ts",
-    "test-vault-example": "ts-node ./examples/vault/index.ts",
     "test-ws-blocks": "ts-node ./examples/websockets/ws-blocks.ts",
     "test-ws-block-headers": "ts-node ./examples/websockets/ws-block-headers.ts",
     "test-ws-filled-blocks": "ts-node ./examples/websockets/ws-filled-blocks.ts",
-    "fix-lint": "tslint -c tslint.json '**/*.ts' --fix"
+    "lint": "eslint .",
+    "lint:fix": "eslint --fix .",
+    "fmt:check": "npm run lint",
+    "fmt": "npm run lint:fix"
```

Comment:
also you might want to write a command to generate the documentation. We did it for typescript package:
`"doc": "typedoc src/index.ts --name massa-web3 --out docs/massa-web3/html --tsconfig tsconfig.json"`